lib/matplotlib/axes/_base.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import (absolute_import, division, print_function,
+﻿from __future__ import (absolute_import, division, print_function,
```

Comment:
Yeah -- we've never deliberately had Unicode BOMs, and since they are all Latin-1 by default (and UTF-8 by our convention if we need them to be), the BOMs aren't necessary.